rocm_getpath.cpp

```diff
@@ -25,10 +25,13 @@
 
 #include <string.h>
 #include <stdlib.h>
-#include <limits.h> /* PATH_MAX */
```

Comment:
put this under condition if not available in windows instead of removing as this is needed based on below comments.